src/libraries/System.Net.Quic/src/System.Net.Quic.csproj

```diff
@@ -41,8 +27,10 @@
   </ItemGroup>
   <!-- Unsupported platforms -->
   <ItemGroup Condition="'$(TargetPlatformIdentifier)' == ''">
-    <Compile Include="System\Net\Quic\QuicListener.Unsupported.cs" />
-    <Compile Include="System\Net\Quic\QuicConnection.Unsupported.cs" />
+    <Compile Include="System\Net\Quic\*.Unsupported.cs" />
+    <!-- [ActiveIssue("https://github.com/dotnet/runtime/issues/71559")]: PNSE generator cannot handle required keyword, so excluding affected classes for now. -->
+    <Compile Include="System\Net\Quic\QuicListenerOptions.cs" />
```

Comment:
I'll improve the comment :smile:. But it "excludes" the classes from automatic PNSE generation (ExcludeApiList.PNSE.txt) and thus it has to include some source code for them manually. I'm using the same code we use on functional platforms, because it's just a property bag and so it's not introducing any code that shouldn't be present on non-functional platforms.